web/themes/joinup/prototype/html-prototype-sandbox/partials/listing-item--event-col-12.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col">
+<div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col mdl-cell--9-col-wide">
```

Comment:
It was not a blocker for the prototype, I was referring to having a template with name for 12 columns and having 9 columns inside or the vice-versa, it's not easy to find going through the file structure if one is not aware of the new size. Please leave it like this, as it can be still found by search, not too problematic.